Automattic/newspack-theme
# Social Menu: Add Podigee icon

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds a Podigee icon to our social menu icon set, as at least one publisher is using it.

Closes #800.

### How to test the changes in this Pull Request:

1. Add a couple links to your social menu -- one using podigee.io (the format I've seen used on one site), and podigee.com (the format I added just in case).
2. Save and publish; view the menu on the front-end - note you have two link icons:

![image](https://user-images.githubusercontent.com/177561/75829215-7176a300-5d62-11ea-9960-b951777b7ed3.png)

3. Apply the PR.
4. Confirm that you have two Podigee icons in your menu:

![image](https://user-images.githubusercontent.com/177561/75829254-8eab7180-5d62-11ea-9072-f3e740655069.png)

### Other information:
